libs/execution/src/lib/transforms/transform-executor.spec.ts

```diff
@@ -134,18 +134,24 @@ describe('Validation of TransformExecutor', () => {
     );
     const transformColumnNames: string[] = ['Column2'];
 
-    const result = await parseAndExecuteTransform(
-      text,
-      inputTable,
-      transformColumnNames,
-    );
+    try {
+      const result = await parseAndExecuteTransform(
+        text,
+        inputTable,
+        transformColumnNames,
+      );
 
-    expect(result.rowsToDelete).toHaveLength(0);
-    expect(result.resultingColumn.valuetype).toEqual(
-      PrimitiveValuetypes.Integer,
-    );
-    expect(result.resultingColumn.values).toHaveLength(1);
-    expect(result.resultingColumn.values).toEqual(expect.arrayContaining([21]));
+      expect(result.rowsToDelete).toHaveLength(0);
+      expect(result.resultingColumn.valuetype).toEqual(
+        PrimitiveValuetypes.Integer,
+      );
+      expect(result.resultingColumn.values).toHaveLength(1);
+      expect(result.resultingColumn.values).toEqual(
+        expect.arrayContaining([21]),
+      );
+    } catch (e) {
+      expect(e).toEqual(undefined);
+    }
```

Comment:
Why do we need that try-catch here?

Reply:
Strictly speaking we don't just added it that all tests have the same format.

Reply:
Then pls remove since it is kind of misleading